packages/sanity/src/structure/panes/document/statusBar/DocumentStatusBar.tsx

```diff
@@ -1,23 +1,34 @@
 import {Flex} from '@sanity/ui'
 import {type Ref, useCallback, useState} from 'react'
-import {useTimelineSelector} from 'sanity'
+import {
+  type CreateLinkMetadata,
+  isCreateLinked,
+  useSanityCreateConfig,
+  useTimelineSelector,
+} from 'sanity'
 
 import {SpacerButton} from '../../../components/spacerButton'
+import {DOCUMENT_PANEL_PORTAL_ELEMENT} from '../../../constants'
 import {useDocumentPane} from '../useDocumentPane'
+import {useDocumentTitle} from '../useDocumentTitle'
 import {DocumentBadges} from './DocumentBadges'
 import {DocumentStatusBarActions, HistoryStatusBarActions} from './DocumentStatusBarActions'
 import {DocumentStatusLine} from './DocumentStatusLine'
 import {useResizeObserver} from './useResizeObserver'
 
 export interface DocumentStatusBarProps {
   actionsBoxRef?: Ref<HTMLDivElement>
+  createLinkMetadata?: CreateLinkMetadata
 }
 
 const CONTAINER_BREAKPOINT = 480 // px
 
 export function DocumentStatusBar(props: DocumentStatusBarProps) {
-  const {actionsBoxRef} = props
-  const {editState, timelineStore} = useDocumentPane()
+  const {actionsBoxRef, createLinkMetadata} = props
+  const {editState, timelineStore, onChange: onDocumentChange} = useDocumentPane()
+  const {title} = useDocumentTitle()
+
+  const CreateLinkedActions = useSanityCreateConfig().components?.documentLinkedActions
```

Comment:
This is done to avoid having to put component code belonging to the Create integration in the structure package.